addon/-private/formatters/-base.ts

```diff
@@ -1,25 +1,39 @@
-/**
- * Copyright 2015, Yahoo! Inc.
- * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
- */
-
-import { warn } from '@ember/debug';
 import { camelize } from '@ember/string';
-import { A as emberArray } from '@ember/array';
+import { warn } from '@ember/debug';
 
 import links from '../../utils/links';
 
 const EMPTY_OBJECT = {};
 
-/**
- * @private
- * @hide
- */
-export default class FormatterBase {
-  get options() {
-    return emberArray();
+export interface FormatterOptions {
```

Comment:
Could be typed per each formatter since they have slightly different shapes/options

Reply:
I like it. Will do.